docs/example-manifests/SLEMicro/dhcp/rke2-control-plane.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - 192.168.0.0/18
+    services:
+      cidrBlocks:
+        - 10.96.0.0/12 # Doesn't matter because they are an internal network
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
+    kind: RKE2ControlPlane
+    name: sample-cluster
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: Metal3Cluster
+    name: sample-cluster
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: Metal3Cluster
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  controlPlaneEndpoint:
+    host: 192.168.125.200 # Has to match the IP of the control plane node
+    port: 6443
+  noCloudProvider: true
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
+kind: RKE2ControlPlane
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: Metal3MachineTemplate
+    name: sample-cluster-controlplane
+  replicas: 1
+  preRKE2Commands:
+    - /usr/lib/systemd/systemd-growfs /var   # this should be removed with https://bugzilla.suse.com/show_bug.cgi?id=1217430
```

Comment:
Can we remove this now?

Reply:
no, we need to keep it as is. The changes added to the EIB part is to remove my fork from OBS with exactly that changes. We can do now into the EIB part instead of using the OBS fork, but we need to keep this line as is to be executed first boot

Reply:
yes, we can removed it. Added a commit to remove it, and it has been tested on the lab successfully without this line

Reply:
Nice! Thanks for checking this.